app/code/Magento/Wishlist/etc/di.xml

```diff
@@ -8,6 +8,8 @@
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
     <preference for="Magento\Wishlist\Model\AuthenticationStateInterface" type="Magento\Wishlist\Model\AuthenticationState" />
     <preference for="Magento\Wishlist\Controller\WishlistProviderInterface" type="Magento\Wishlist\Controller\WishlistProvider" />
+    <preference for="Magento\Wishlist\Api\WishlistRepositoryInterface" type="Magento\Wishlist\Model\WishlistRepository" />
```

Comment:
Preference for WishlistInterface is missing

Reply:
Added